cysync/src/errors/helpers.ts

```diff
@@ -114,8 +114,114 @@ const handleWalletErrors = (
     );
 };
 
-// const handleUnknownFlowErrors;
-// const;
+export const getMap = (langStrings: I18nStrings): CodeToErrorMap => {
+  return {
+    [DeviceErrorType.CONNECTION_CLOSED]: {
+      parent: DeviceErrorType.DEVICE_DISCONNECTED_IN_FLOW,
+      message: 'Device connection closed'
+    },
+    [DeviceErrorType.CONNECTION_NOT_OPEN]: {
+      parent: DeviceErrorType.DEVICE_DISCONNECTED_IN_FLOW,
+      message: 'Device connection not open'
+    },
+    [DeviceErrorType.DEVICE_DISCONNECTED_IN_FLOW]: {
+      message: langStrings.ERRORS.DEVICE_DISCONNECTED_IN_FLOW
+    },
```

Comment:
An example of error code to message mapping.
In this particular case `CONNECTION_CLOSED` & `CONNECTION_NOT_OPEN` are child errors those are not displayed to the UI. the handler climbs up the object tree to find its parent and set the display message in I18n

NOTE: Currently all internal errors are hardcoded in English as it never sees the UI, so dont mind it.